03 Navigation/src/components/app.tsx

```diff
@@ -1,13 +1,8 @@
 import * as React from 'react';
-import * as ReactDOM from 'react-dom';
-import { Router, Route, IndexRoute, Link, IndexLink, browserHistory, hashHistory  } from 'react-router'
+import Header from './common/header';
 
-import Header from './common/header'
-import aboutPage from './about/aboutPage';
-import membersPage from './members/membersPage';
 
-
-interface Props {
+interface Props extends React.Props<App> {
   children : any;
```

Comment:
remove this property.
When you extends from React.Props you do already have this property.